kasa/cli.py

```diff
@@ -629,7 +642,7 @@ async def emeter(dev: SmartDevice, year, month, erase):
         usage_data = await dev.get_emeter_daily(year=month.year, month=month.month)
     else:
         # Call with no argument outputs summary data and returns
-        emeter_status = dev.emeter_realtime
+        emeter_status = await dev.get_emeter_realtime()
```

Comment:
This will cause extra I/O request even when not necessary (i.e., when no index/name is specified). Maybe check if either index or name is specified just to avoid changing the earlier behavior?